Sources/Shared/VimeoClient.swift

```diff
@@ -159,7 +176,11 @@ final public class VimeoClient {
      
      - returns: a `RequestToken` for the in-flight request
      */
-    public func request<ModelType>(_ request: Request<ModelType>, completionQueue: DispatchQueue = DispatchQueue.main, completion: @escaping ResultCompletion<Response<ModelType>, NSError>.T) -> RequestToken {
+    public func request<ModelType>(
+        _ request: Request<ModelType>,
+        completionQueue: DispatchQueue = DispatchQueue.main,
```

Comment:
Nit: thoughts on just doing:
```suggestion
        completionQueue: DispatchQueue = .main,
```